docs/data/data-grid/clipboard/ClipboardCopy.js

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { DataGridPremium, useGridApiRef } from '@mui/x-data-grid-premium';
+import { useDemoData } from '@mui/x-data-grid-generator';
+import Alert from '@mui/material/Alert';
+import AlertTitle from '@mui/material/AlertTitle';
+
+export default function ClipboardCopy() {
+  const { data } = useDemoData({
+    dataSet: 'Commodity',
+    rowLength: 10,
+    maxColumns: 20,
+  });
+
+  const [copiedData, setCopiedData] = React.useState('');
+
+  const apiRef = useGridApiRef();
+
+  React.useEffect(() => {
+    return apiRef.current.subscribeEvent('clipboardCopy', (copiedString) => {
+      setCopiedData(copiedString);
+    });
+  }, [apiRef]);
+
+  return (
+    <div style={{ width: '100%' }}>
+      <div style={{ height: 400 }}>
+        <DataGridPremium
+          apiRef={apiRef}
+          {...data}
+          checkboxSelection
+          disableRowSelectionOnClick
+          unstable_cellSelection
```

Comment:
Given that the Clipboard paste is now using the same API for persistence as Editing, maybe we should enable it by default (as it only impacts the editable cells)?
I can add an experimental flag for it instead of `unstable_cellSelection`